crates/macros/src/bitfields/utils.rs

```
@@ -0,0 +1,66 @@
+}
+
+pub fn type_from_size(size: usize) -> syn::Result<Box<TypePath>> {
+    match size {
+        1 => Ok(parse_quote!(bool)),
+        2..=8 => Ok(parse_quote!(u8)),
+        9..=16 => Ok(parse_quote!(u16)),
+        17..=32 => Ok(parse_quote!(u32)),
+        33..=64 => Ok(parse_quote!(u64)),
+        65..=128 => Ok(parse_quote!(u128)),
+        _ => {
+            return Err(syn::Error::new_spanned(
```

Clippy (1.95.0)

[warning] unneeded `return` statement

(warning)